tensorflow/compiler/xla/service/gpu/cudnn_conv_runner.cc

```diff
@@ -82,6 +82,7 @@ Status RunCudnnConvImpl(const CudnnConvParams& params,
   auto output_buf = se::DeviceMemory<T>(params.output_buf);
   AlgorithmConfig algorithm = params.algorithm;
 
+#if TENSORFLOW_USE_ROCM
   if (options.first_call_from_algorithm_picker) {
```

Comment:
[TODO] Remove the `#if` and use stream_executor to check for system arch instead